File: Python3/excel_sheet_column_title.py
class Solution:
    def convertToTitle(self, n: int) -> str:
        title = ""
        
        while n > 0:
            index = (n-1) % 26
            title = chr(ord("A") + index) + title
            n = (n-1) // 26
        
        return title

File: C/excel_sheet_column_title.c
char * convertToTitle(int n){
    char* title = malloc(100 * sizeof(char));
    int index = 0;
        
    while(n > 0) {
        title[index] = (char)('A' + ((n-1) % 26));
        n = (n-1) / 26;
        index ++;
    }
    title[index] = '\0';
    
    for (int i = 0; i < index / 2; i++) {
        char temp = title[index - i - 1];
        title[index - i - 1] = title[i];
        title[i] = temp;
    }
    
    return title;
}
